### PR TITLE
webui: Feature #5136, Add Filmaffinity search in broadcast details

### PR DESCRIPTION
**Pull request by pablozg**
I added the request feature #5136, if you think is usefull, please merge it.

Thanks.

### CODE REVIEW

File: src/webui/static/app/dvr.js [hunk 1]
```diff
@@ -119,27 +119,41 @@ tvheadend.dvrDetails = function(uuid) {
...
         }
 
-        function searchTheTVDB(){
+        function searchTheTVDB() {
             window.open('http://thetvdb.com/?string='+
                         encodeURIComponent(title)+'&searchseriesid=&tab=listseries&function=Search','_blank');
         }
 
+        function searchFilmAffinity() {
+            window.open('https://www.filmaffinity.com/es/search.php?stext=' +
```
Comment (CvH) [inline src/webui/static/app/dvr.js]: is it a good choice to hard coding the spanish version ?

Comment (pablozg) [reply src/webui/static/app/dvr.js]: My fault, I will add a language check to use the English or Spanish version.

Comment (mpmc) [reply src/webui/static/app/dvr.js]: IMO, I would just use en & not bother with any checks. The reasons for this are..

- People will ask  "why isn't <insert lang here> supported in tvheadend, it's supported on the site."
- The available languages (on the site) may change at any point.

Comment (pablozg) [reply src/webui/static/app/dvr.js]: Filmaffinity it's a spanish page, it's support two languages (english and spanish), by default now it's locale dependent, so will use spanish version only if you set it as language in user config.

**Comment by perexg** [comment]
Could we use a combobox to select the search type? It seems that we end with a long bar of search icons...

**Comment by pablozg** [comment]
OK. I'll take a look.

**Pull request closed without merging**